src/graphics/prefilter-cubemap.js

```diff
@@ -483,4 +485,62 @@ function shFromCubemap(source, dontFlipX) {
     return sh;
 }
 
-export { prefilterCubemap, shFromCubemap };
+function prefilterCubemap2(cubemap) {
```

Comment:
I think we need to think of a better name here. Can we deprecate the original `prefilterCubemap` function and introduce a nicely, appropriately named replacement?